From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <samuel.r.attard@gmail.com>
Date: Wed, 8 Feb 2023 02:54:05 -0800
Subject: test: formally mark some tests as flaky

Instead of disabling the tests, flag them as flaky so they still run
but don't cause CI failures on flakes.

diff --git a/test/parallel/parallel.status b/test/parallel/parallel.status
index 6303908ce180db4d409707ae1f049319358642fb..d487497417e20c778dde57197dc18373799df36a 100644
--- a/test/parallel/parallel.status
+++ b/test/parallel/parallel.status
@@ -5,6 +5,16 @@ prefix parallel
 # sample-test                        : PASS,FLAKY
 
 [true] # This section applies to all platforms
+# https://github.com/nodejs/node/pull/50327
+# Currently there's no reliable way to test it.
+test-diagnostics-channel-memory-leak: SKIP
+test-inspector-multisession-ws: PASS, FLAKY
+test-heapsnapshot-near-heap-limit-by-api-in-worker: PASS, FLAKY
+test-heapsnapshot-near-heap-limit-worker: PASS, FLAKY
+test-fetch: PASS, FLAKY
+test-cluster-bind-privileged-port: PASS, FLAKY
+test-cluster-shared-handle-bind-privileged-port: PASS, FLAKY
+test-debugger-random-port-with-inspect-port: PASS, FLAKY
 # https://github.com/nodejs/node/issues/52273
 test-net-write-fully-async-hex-string: PASS, FLAKY
 # https://github.com/nodejs/node/issues/52273
diff --git a/test/sequential/sequential.status b/test/sequential/sequential.status
index 6893abfe6e1c03d455099099f0d7e6b6930372a6..24aab3bad500ce4b9d43bf693c5efa6d563fe858 100644
--- a/test/sequential/sequential.status
+++ b/test/sequential/sequential.status
@@ -7,6 +7,18 @@ prefix sequential
 [true] # This section applies to all platforms
 # https://github.com/nodejs/node/issues/27611#issuecomment-613100468
 test-cpu-prof-dir-worker: PASS, FLAKY
+test-cpu-prof-dir-and-name: PASS, FLAKY
+test-cpu-prof-drained: PASS, FLAKY
+test-cpu-prof-default: PASS, FLAKY
+test-cpu-prof-dir-absolute: PASS, FLAKY
+test-cpu-prof-dir-relative: PASS, FLAKY
+test-cpu-prof-exit: PASS, FLAKY
+test-cpu-prof-kill: PASS, FLAKY
+test-cpu-prof-name: PASS, FLAKY
+test-cpu-prof-worker-argv: PASS, FLAKY
+test-diagnostic-dir-cpu-prof: PASS, FLAKY
+test-worker-prof: PASS, FLAKY
+test-inspector-stop-profile-after-done: PASS, FLAKY
 # https://github.com/nodejs/node/issues/44898
 test-watch-mode: PASS, FLAKY
 test-watch-mode-inspect: PASS, FLAKY
